pkg/karmadactl/describe.go

```diff
@@ -39,17 +35,19 @@ var (
 
 // NewCmdDescribe new describe command.
 func NewCmdDescribe(karmadaConfig KarmadaConfig, parentCommand string) *cobra.Command {
-	ioStreams := genericclioptions.IOStreams{In: getIn, Out: getOut, ErrOut: getErr}
-	o := &CommandDescribeOptions{
-		FilenameOptions: &resource.FilenameOptions{},
-		DescriberSettings: &describe.DescriberSettings{
-			ShowEvents: true,
-			ChunkSize:  cmdutil.DefaultChunkSize,
+	streams := genericclioptions.IOStreams{In: getIn, Out: getOut, ErrOut: getErr}
```

Comment:
We can rebase this branch after #2164 merged.

Reply:
Yes, I'll deal with it tonight @lonelyCZ